libgalois/include/katana/PropertyGraph.h

```diff
@@ -694,6 +709,30 @@ class KATANA_EXPORT PropertyGraph {
     return edge_entity_type_manager_;
   }
 
+  /// Get the version from the RDGFile
+  katana::RDGVersion RDGFileVersion() {
+    if (file_ == nullptr) {
+      return katana::RDGVersion(0);
+    } else {
+      return rdg_.GetFileVersion(*file_);
+    }
+  }
+
+  /// Check if the loaded PropertyGraph is intermediate
+  bool IsIntermediate() { return intermediate_; }
+
+  /// Set if the loaded PropertyGraph is intermediate
+  void SetIntermediate(bool val) { intermediate_ = val; }
+
+  /// Set the version of loaded PropertyGraph
+  void SetLoadedVersion(const katana::RDGVersion& val) {
+    loaded_version_ = val;
+  }
+  /// Get the version of loaded PropertyGraph (const)
+  const katana::RDGVersion& GetLoadedVersion() const { return loaded_version_; }
```

Comment:
Why is this the "loaded version" instead of just the "version"? It is because the graph may have been changed since it was loaded, so the version may be out of date (and need to be updated during write)? Please explain what distinguishes the "version" from the "loaded version" in the doxygen comment.